be/src/olap/options.h

```diff
@@ -26,11 +26,14 @@
 namespace doris {
 
 struct StorePath {
-    StorePath() : capacity_bytes(-1) { }
+    StorePath() : capacity_bytes(-1), storage_medium(TStorageMedium::HDD){ }
```

Comment:
```suggestion
    StorePath() : capacity_bytes(-1), storage_medium(TStorageMedium::HDD) { }
```

Reply:
How do you format the code? I can't find .clang-format, is it a predefined style?

Reply:
The code in Doris comes from many different projects. Generally, the style of each file is consistent, if you add code to a file, you should keep style with it. If it is a new file, Baidu's code style is used. It's about the same style as Google's. Except lowercase and underscore as the function name, and use 4-byte indentation and so on.

clang-format is a good suggestion, we should introduce it to our project to make developers easier.